eternalcore/src/main/java/com/eternalcode/core/chat/feature/privatechat/PrivateChatService.java

```diff
@@ -20,21 +22,31 @@ public class PrivateChatService {
     private final IgnoreRepository ignoreRepository;
     private final Publisher publisher;
     private final UserManager userManager;
+    private final Server server;
 
     private final Cache<UUID, UUID> replies = CacheBuilder.newBuilder()
         .expireAfterWrite(Duration.ofHours(1))
         .build();
 
     private final Set<UUID> socialSpy = new HashSet<>();
 
-    public PrivateChatService(NoticeService noticeService, IgnoreRepository ignoreRepository, Publisher publisher, UserManager userManager) {
+    public PrivateChatService(NoticeService noticeService, IgnoreRepository ignoreRepository, Publisher publisher, UserManager userManager, Server server) {
```

Comment:
nie musisz brać całego serwera żeby to sprawdzić, wystarczy dodać do `ClientSettings` is online